doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,21 +117,25 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
 To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
 
-    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Secure --serial-number <serial_number>
+    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Application --serial-number <serial_number>
 
 You need to follow this flashing process only one time, as the PCB configuration will not change.
 
 Programming the SDFW and SCFW
 =============================
 
-After programming the BICR, the nRF54H20 SoC requires the provisioning of a bundle ( :file:`nrf54h20_soc_binaries_v0.3.3.zip`) containing the precompiled firmware for the Secure Domain and System Controller.
+##TODO
+
+After programming the BICR, the nRF54H20 SoC requires the provisioning of a bundle ( :file:`nrf54h20_soc_binaries_v0.5.0.zip`) containing the precompiled firmware for the Secure Domain and System Controller.
 To program the Secure Domain Firmware (SDFW, also known as ``urot``) and the System Controller Firmware (SCFW) from the firmware bundle to the nRF54H20 DK, do the following:
 
-1. Download the `nRF54H20 firmware bundle`_.
+1. Download the `nRF54H20 firmware bundle v0.5.0`_.
 #. Move the :file:`ZIP` bundle to a folder of your choice.
 #. |open_terminal_window_with_environment|
 #. Run nRF Util to program the binaries using the following command::
```

Comment:
Is there an easy way to verify the current LCS? @tjora

Reply:
Would this command work in both empty and rot? @tjora nrfutil device x-suit-dfu --firmware <path-to_bundle_zip_file> --serial-number <serialnumber>

Reply:
The command would be identical, but you are using `--core Application` rather than `--core Secure`.

Note that I do not think we have done much testing on how to perform this replacement in ROT, so I am not sure its something we should recommend in the docs.

@David-Garcia-Polo suggested something like this, as the safe option (1):
1. Erase the existing BICR
2. Modify the board
3. Program the new BICR
4. Reset

Another one that would probably work (2):
1. Program the new BICR
2. Power of the device _before any resets_
3. Modify the board
4. Reset

If you do not need to modify the board, and just need to patch some values then this would be fine(3):
1. Program the new BICR
2. Reset

Strategy (1) would require that we test that sdsc will allow us to reprogram the chip if BICR is not programmed.
Strategy (2) would require that we have a lot of confidence that that are no undue resets between programming and powering of the board. Personally I think this is quite scary.
Strategy (3) should be fine, but I do not know how well this is tested by SysFW devs.

Reply:
The `x-suit-dfu` command will only work in ROT @FrancescoSer , as the DFU procedure is performed by talking to present and running sdfw, which only happens in ROT.

Reply:
Should these steps also assume that users might want to upgrade the bundle on an existing custom board? Also is there a scenario where the user might want to update also the BICR? (if yes, how?) @ingeryp @tjora @David-Garcia-Polo

Reply:
## Update BICR:
Yes, customer may want to do this if they make modifications to their PCB, such as supplying an external square wave rather than the on-board oscillator, to tweak certain values, or to change the power supply mode.

You can reprogram the BICR in LCS ROT+ using this command:
`nrfutil device program --firmware <bicr.hex> --core Application --serial-number <snr>`
However the exact sequencing is not clear, and most importantly _not tested_.
See this comment https://github.com/nrfconnect/sdk-nrf/pull/16796#discussion_r1745303511

## Update SDSC
Yes, I would assume customers would need to do this. This can be done with the following commands
```
unzip nrf54h20_soc_binaries_v<x.y.z>.zip
nrfutil device x-suit-dfu --serial-number $snr --firmware nordic_top.suit
```
You can verify the upgrade by running this command before and after you upgrade sdsc. The version number should increase after the dfu procecure is complete:
```
nrfutil device x-sdfw-version-get --firmware-slot uslot --serial-number <snr>
```
@hakonfam could expand on this if needed.

Reply:
If the BICR update sequence is not tested, I'll not document it right now

Reply:
Added the update section @ingeryp